From 9a2803e00f1f10e00376b46366cd290a8ff55875 Mon Sep 17 00:00:00 2001
From: "Manjunathappa, Prakash" <prakash.pm@ti.com>
Date: Tue, 6 Dec 2011 19:03:49 +0530
Subject: [PATCH 0647/1092] video: da8xx-fb: Add debug message on sync and
 underflow error

Patch adds useful debug message on LCDC syn and underflow error.

Signed-off-by: Manjunathappa, Prakash <prakash.pm@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 drivers/video/da8xx-fb.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/video/da8xx-fb.c b/drivers/video/da8xx-fb.c
index 7bb1c16..8c50985 100644
--- a/drivers/video/da8xx-fb.c
+++ b/drivers/video/da8xx-fb.c
@@ -769,6 +769,7 @@ static irqreturn_t lcdc_irq_handler_rev02(int irq, void *arg)
 	u32 reg_int;
 
 	if ((stat & LCD_SYNC_LOST) && (stat & LCD_FIFO_UNDERFLOW)) {
+		printk(KERN_ERR "LCDC sync lost or underflow error occured\n");
 		lcd_disable_raster();
 		clk_disable(par->lcdc_clk);
 		lcdc_write(stat, LCD_MASKED_STAT_REG);
@@ -832,6 +833,7 @@ static irqreturn_t lcdc_irq_handler_rev01(int irq, void *arg)
 	u32 reg_ras;
 
 	if ((stat & LCD_SYNC_LOST) && (stat & LCD_FIFO_UNDERFLOW)) {
+		printk(KERN_ERR "LCDC sync lost or underflow error occured\n");
 		lcd_disable_raster();
 		clk_disable(par->lcdc_clk);
 		lcdc_write(stat, LCD_STAT_REG);
-- 
1.7.11.2

